metrics/density/footprint_data.md

```diff
@@ -0,0 +1,104 @@
+# Footprint data script details
+
+The `footprint_data.sh` test within this directory can be configured with a number
+of environment variables.
+
+Details of the configurable variables are listed below, along with a small example
+invocation script.
+
+## Variables
+The script action can be modified by setting a number of environment variables.
+
+These variables take effect generally in two ways. Some variables affect how the
+payload is executed. The `RUNTIME` and `PAYLOAD` arguments directly affect the
+payload execution via this line in the script:
+
+`docker run -tid --runtime=$RUNTIME $PAYLOAD_RUNTIME_ARGS $PAYLOAD $PAYLOAD_ARGS`
+
+Other settings affect how memory footprint is measured and the test termination
+conditions.
+
+| Variable | Function
+| -------- | --------
+| PAYLOAD | The docker image to run
+| PAYLOAD_ARGS | Any arguments passed into the docker image
```

Comment:
Could you mention in the brief intro that this script only works for the docker container manager.